src/seedu/addressbook/AddressBook.java

```diff
@@ -260,17 +268,18 @@ private static void echoUserCommand(String userCommand) {
      *
      * @param args full program arguments passed to application main method
      */
+    
     private static void processProgramArgs(String[] args) {
-        if (args.length >= 2) {
+        if (args.length >= LENGTH_INVALID_PROGRAM) {
```

Comment:
having arg length >=2 doesn't make the program invalid. in light of this argument, you may want to choose a better variable name for this one